### PR TITLE
[#IOCIT-9, #IOCIT-11] App version on profile update and retrieve

### PR DESCRIPTION
**Pull request by michaeldisaro**
This PR depends on #897.

#### List of Changes

Update of openapi specs and ts-commons library to include AppVersion on Profile.

#### Motivation and Context

We need to save and retrieve AppVersion with Profile to know which users are not able to opt-out from premium messages reading status sharing because of an old app.

#### How Has This Been Tested?

Not tested yet. Another PR is locking this and build will fail until a merge will be done.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/controllers/__tests__/profileController.test.ts [hunk 1]
```diff
@@ -56,7 +55,8 @@ const proxyUserResponse = {
   preferredLanguages: aPreferredLanguages,
   spid_email: anEmailAddress,
   token: "123hexToken",
-  version: 1 as NonNegativeInteger
+  version: 1 as NonNegativeInteger,
+  last_app_version: "UNKNOWN"
```
Comment (BurnedMarshal) [inline src/controllers/__tests__/profileController.test.ts:59]: `last_app_version` should never be equals to `UNKNOWN`. Or is a valid `Semver` or is missing.

File: src/controllers/__tests__/profileController.test.ts [hunk 2]
```diff
@@ -65,7 +65,8 @@ const apiUserProfileResponse = {
   is_inbox_enabled: true,
   is_webhook_enabled: true,
   preferred_languages: ["it_IT"],
-  version: 42
+  version: 42,
+  last_app_version: "UNKNOWN"
```
Comment (BurnedMarshal) [inline src/controllers/__tests__/profileController.test.ts:69]: ditto

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]

File: src/controllers/__tests__/profileController.test.ts (hunk 1 above)
Comment (michaeldisaro) [reply src/controllers/__tests__/profileController.test.ts]: https://github.com/pagopa/io-backend/pull/903/commits/4552552ce849651c5835f3e4d4a9c62fa0338427

File: src/controllers/__tests__/profileController.test.ts (hunk 2 above)
Comment (michaeldisaro) [reply src/controllers/__tests__/profileController.test.ts]: https://github.com/pagopa/io-backend/pull/903/commits/4552552ce849651c5835f3e4d4a9c62fa0338427

**Review by BurnedMarshal — APPROVED** [review]
LGTM

**Pull request merged**